msteams-platform/concepts/build-and-test/analyze-your-apps-usage-in-developer-portal.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Analyze your app's usage in Developer Portal
+description: In this module, learn how to analyze your app's usage in Developer Portal
+ms.localizationpriority: medium
+ms.topic: overview
+ms.author: surbhigupta
+---
+
+# Analyze your app's usage in Developer Portal
+
+In the Developer Portal for Teams, on the **Overview** page, you can see the total number of active users for your app.
+
+> [!NOTE]
+> Usage analytics are currently available only for new custom apps published to your org through **Developer Portal** for Teams or imported into **Developer Portal** for Teams after April 2022. Usage analytics for all the apps published to the Teams store is available in **Partner Center**, for more information [Teams apps usage report](/office/dev/store/teams-apps-usage).
+
+| Metric | Definition |
+| :-----------------------| :------------------------------------------------------------------------------------------------------|
+| *Monthly R30* | The default usage metric. It shows the count of unique active users who used your app within that rolling 30 day window in UTC. |
+| *Daily* | It shows you the count of unique active users who used your app in a given day in UTC. |
+
+The app usage for a given day is reflected within 24 to 48 hours, and usage data for new apps can take up to three to five days to reflect in the charts.
+
+You can view your app's usage and other insights from the **Analytics** page. To access the page:
+
+1. Go to **[Developer Portal for Teams](https://dev.teams.microsoft.com)**.
+1. Select **Apps** from the left pane.
+1. Select the required app from the **Apps** page.
+1. Select **Analytics** under the **Overview** or select **View details** under the **Active Users (Preview)** card.
+
+ :::image type="content" source="../../assets/images/tdp/dev-app-portal.PNG" alt-text="The screenshots shows you the analytics page of your app in Developer Portal."lightbox="../../assets/images/tdp/dev-app-portal.PNG":::
+
+As you explore individual metrics on this page, you can use the **Filter** button to analyze your app's usage from the following filter options:
+
+* Aggregation type: This filter allows you to group the following metrics by a count of distinct users or a count of distinct tenants or customers.
+* Platform
+* Operating system
+* Area
+
+ :::image type="content" source="../../assets/images/tdp/dev-analytics-filter.PNG" alt-text="The screenshots shows you the analytics page filter in Developer Portal.":::
```

Comment:
filename extn was changed to lowercase. While this mention will also work (our build pipeline handles mixed casing gracefully), but it'd be better to use lowercase even in the links/references to the files.